src/providers/twitch/TwitchChannel.cpp

```diff
@@ -321,13 +321,13 @@ void TwitchChannel::addPartedUser(const QString &user)
         this->partedUsersMergeQueued_ = true;
 
         QTimer::singleShot(500, &this->lifetimeGuard_, [this] {
-            auto partedUsers = this->partedUsers_.access();
```

Comment:
It is always wrong to capture the partedUsers in the lambda so why not reuse the variable name?